src/main/java/uk/gov/hmcts/reform/bulkscan/orchestrator/services/ccd/CcdCaseUpdater.java

```diff
@@ -328,4 +326,15 @@ private CaseDataContent buildCaseDataContent(
             .build();
     }
 
+    class Credentials {
+        final String idamToken;
+        final String s2sToken;
+        final String userId;
+
+        Credentials(String idamToken, String s2sToken, String userId) {
+            this.idamToken = idamToken;
+            this.s2sToken = s2sToken;
+            this.userId = userId;
+        }
+    }
```

Comment:
erm. that falls into another ticket - BPS-1074: refactor CcdCaseUpdater and its surroundings

Reply:
As I explained in the PR channel I have had to introduce a wrapper class to reduce the number of method parameters (to pass the quality gateway), just using `startEvent.getCaseDetails().getId()` instead of `existingCaseId` caused a failure in functional test

Reply:
can use suppression flag for that with the note to be addressed in future prs as it is pending/planned.

the other note is just reverting back the change introduced earlier in pr. which is good

Reply:
This is a change in the right direction, but I think that when we introduce it, we should wrap credentials in a class as early as possible, i.e. before even calling this class. Worth creating a separate request for that (perhaps as the first one to be merged).